src/views/issues.graphql

```diff
@@ -1,5 +1,5 @@
-query Issues($searchQuery: String!) {
-  search(first: 100, type: ISSUE, query: $searchQuery) {
+query Issues($searchQuery: String!, $type: SearchType = ISSUE) {
```

Comment:
Do we still need https://github.com/mozilla-frontend-infra/codetribute/pull/169/files#diff-fd6b3e26da5d99c15109f388dc828f31R7 now that we narrowed down the SearchType to ISSUE?

Reply:
i think we still need it when there is ... in the query. we still query for Repository first, its just that I set the searchType default value to ISSUE. Maybe i need a better naming for the query to indicate it